pkg/cmd/util/tokencmd/request_token.go

```diff
@@ -49,7 +49,7 @@ type RequestTokenOptions struct {
 func RequestToken(clientCfg *restclient.Config, reader io.Reader, defaultUsername string, defaultPassword string) (string, error) {
 	handlers := []ChallengeHandler{}
 	if GSSAPIEnabled() {
-		handlers = append(handlers, NewNegotiateChallengeHandler(NewGSSAPINegotiator()))
+		handlers = append(handlers, NewNegotiateChallengeHandler(NewGSSAPINegotiator(defaultUsername)))
```

Comment:
You could pass in the password as well and emit a warning that it was ignored if Kerberos was used (assuming a password was also given).